### DISCUSSION TITLE
不知道该怎么配置性格什么的

### DISCUSSION THREAD

**Discussion by youheiheia** [TBXark/ChatGPT-Telegram-Workers#159] [Q&A] [answered]
原谅我是小白，一点都不会编程，目前跟着教程已经配置成功可以聊天了，但是不知道怎么配置性格，我搜了很久找到了你在v2ex的教程，用SETENV SYSTEM_INIT_MESSAGE=这个命令设置了一个性格，但是不太了解SETENV KEY=VALUE |KEY|说明|例子| |--|--|--| |SYSTEM_INIT_MESSAGE|这个命令要怎么使用，我给机器人发过去它给我解释了一下这个但是具体使用还是不太懂，是要在说明和例子那里面填上设定吗，其他的都不用动在发过去就行吗。
还有就是设置自动更新的内容不太理解，我查到要申请github的apikey，但是创建的时候要选哪些使用选项可以告诉一下吗，然后就是同步仓库那一步也不太懂（大哭）
如果能有空回我解答的话就太感谢了。

**Comment by tbxark** [comment, marked as answer]
版本更新了。

现在用法是`/setenv SYSTEM_INIT_MESSAGE=这个命令设置了一个性格`

**Reply by youheiheia** [reply]
请问这个配置了开新对话要重新设置吗，用这个命令是只能配置性格还是能把设定写进去，比如名字，爱好什么的。如果要添加设定的话是要把之前的也再写一遍还是只写新的就可以。

**Reply by tbxark** [reply]
写一次就可以。每次修改都会覆盖。

**Comment by tbxark** [comment]
自动更新这个说实话，挺复杂的。现在更新的不太频繁，我建议可以用手动更新。

**Reply by youheiheia** [reply]
作者大大，我用这个命令设置了性格开新对话就会变白纸重新开始

**Comment by wei-haoshuai** [comment]
请问这个怎么弄呀，看不懂
是需要下载那些东西呢？

**Reply by youheiheia** [reply]
我也是小白，你可能回错了😂😂，我是跟着那个视频搞得，然后看的大佬的部署流程，就在开头那个说明里，有个readme，点开跟着步骤走就行
